tools/flang2/flang2exe/ll_write.c

```diff
@@ -1308,6 +1308,8 @@ get_metadata_name(enum LL_MDName name)
     return "!llvm.module.flags";
   case MD_llvm_dbg_cu:
     return "!llvm.dbg.cu";
+  case MD_llvm_linker_options:
+    return "!llvm.linker.options";
```

Comment:
Is this supported in llvm 3.9 as well? Looks like https://reviews.llvm.org/D31349 is pretty new

Reply:
Nope. Silly me.

Reply:
That's fine. If it works for 4.0 then that's fine with me. We can work on 3.9 later

Reply:
It doesn't work for 4.0. It works for 5.0. For 4.0, you have this:

```
!0 = !{i32 6, !"Linker Options", !1}
!1 = !{!2, !3}
!2 = !{!"/DEFAULTLIB:libcmtd.lib"}
!3 = !{!"/DEFAULTLIB:oldnames.lib"}
```

Reply:
Shall I have another flag for `flang2.exe` for llvm version?